package.json

```diff
@@ -261,7 +261,7 @@
     "@metamask/rpc-methods-flask": "npm:@metamask/rpc-methods@0.37.1-flask.1",
     "@metamask/safe-event-emitter": "^2.0.0",
     "@metamask/scure-bip39": "^2.0.3",
-    "@metamask/signature-controller": "^4.0.1",
```

Comment:
There was patch on `@metamask/signature-controller`. I believe that it is still needed, right @matthewwalsh0?
Either way, if it is still required, you can remove and re-add a patch to the v5.3.0. If it is not required anymore, you can remove the patch